pkg/trace/config/config.go

```diff
@@ -116,6 +116,18 @@ type AgentConfig struct {
 
 	// Obfuscation holds sensitive data obufscator's configuration.
 	Obfuscation *ObfuscationConfig
+
+	// RequireTags represents a list of tags required for a root span.
+	RequireTags []*Tag
+
+	// RejectTags represents a list of tags that a root span must not have.
+	RejectTags []*Tag
+}
+
+// Tag represents a key/value pair for filterable tags.
```

Comment:
```suggestion
// Tag represents a key/value pair.
```
It can be for anything 🙂 We document what a structure is on it's own, not the context where it is used.